src/style.css

```
@@ -1 +1,60 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 12px;
+  font-family: "Roboto" , sans-serif;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.img__logo {
+  margin-top: 5px;
+}
+
+.menu {
+  display: flex;
+}
+
+.menu__item {
+  position: relative;
+  text-transform: uppercase;
+  font-weight: 500;
+}
+
+.menu__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.menu__list {
```

list is a bit confusing name for a class that you use on a link
```suggestion
.menu__link {
```